src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -20,42 +27,73 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book Management", description = "Endpoints for managing books")
 @RestController
 @RequestMapping("/api/books")
 @AllArgsConstructor
 public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public ResponseEntity<SuccessResponse<List<BookDto>>> getAll() {
+    @Operation(summary = "Get all books", description = "Get all books")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", useReturnTypeSchema = true)
+    })
+    public SuccessResponse<List<BookDto>> getAll(Pageable pageable) {
```

Comment:
Would be nice to just set the `@ResponseStatus(OK)` and return `List<BookDto>` right away instead of wrapping them to `SuccessResponse` in code. Same for other methods. IMHO looks cleaner that way and there is no real reason to use `ResponseEntity` unless we want to have different types/statuses etc. based on some user input in that endpoint which is kind of a rare situation.
All of that is not necessary, just a suggestion though :wink: